include/bluetooth/services/ots.h

```diff
@@ -33,6 +33,9 @@ extern "C" {
 /** @brief Length of OTS object ID string (in bytes). */
 #define BT_OTS_OBJ_ID_STR_LEN 15
 
+/** @brief ID of the Directory Listing Object */
+#define OTS_OBJ_ID_DIR_LIST     0x000000000000
```

Comment:
Why do we need to expose this information to the user? I think we can move this definition to the internal header as well

Reply:
I guess the idea was that the object ID would be used for a potential client, but we can revisit that once we decide to add that. Moved to internal for now.